python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,92 @@
-| 786         | 757                                      | 747                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 809         | 779                                      | 769                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 832         | 801                                      | 791                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 855         | 823                                      | 813                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 877         | 844                                      | 834                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 900         | 866                                      | 856                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 923         | 888                                      | 877                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 945         | 909                                      | 898                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 968         | 931                                      | 920                                  |
-+-------------+------------------------------------------+--------------------------------------+
-| 991         | 953                                      | 942                                  |
-+-------------+------------------------------------------+--------------------------------------+
-
-:math:`\widehat{q}_{Y^i}(\alpha)` is often called the “empirical
-:math:`\alpha`-quantile” for the variable :math:`{Y^i}`.
+    F_{\sampleSize, \alpha}(k-1)\geq \beta
+
+The smallest rank :math:`k_{sol}` such that the previous equation is satisfied is:
+
+.. math::
+
+    k_{sol} & = \min \{ k \in \llbracket 1, n \rrbracket \, | \, F_{\sampleSize, \alpha}(k-1)\geq \beta \}\\
+            & = 1 +  \min \{ k \in \llbracket 1, n\rrbracket] \, | \, F_{\sampleSize, \alpha}(k)\geq \beta \}
+
+An upper bound of  :math:`x_{\alpha}` is estimated by the value of :math:`X_{(k_{sol})}`  on the sample
+:math:`(x_1, \dots, x_\sampleSize)`.
+
+Minimum sample size for an upper bound to the quantile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Given :math:`\alpha`, :math:`\beta`, and :math:`k`, we seek for the smallest sample size :math:`\sampleSize`
+such that the equation :eq:`EqOrderStat` is satisfied. Then the :eq:`EqOrderStat2` is solved with respect to the rank :math:`\sampleSize`.
```

Comment:
Sorry, I did not understand the meaning of the sentence and was confused by the fact that the text mentions two different equations. Let me please rephrase this.
```suggestion
Given :math:`\alpha`, :math:`\beta`, and :math:`k`, we seek for the smallest sample size :math:`\sampleSize`
such that the equation :eq:`EqOrderStat` is satisfied. In order to do so, we solve the equation :eq:`EqOrderStat2` with respect to the sample size :math:`\sampleSize`.
```

Reply:
ok!